excercises/02. Function Exercises/01. Calculating Average/Calculate_average_Morme_02.py

```diff
@@ -0,0 +1,16 @@
+def getNumber1():
+    return int(input("Podaj pierwszą liczbę: "))
+
+def getNumber2():
+    return int(input("Podaj drugą liczbę: "))
+
+def getNumber3():
```

Comment:
Zobacz, te funkcje są bardzo podobne. Może możnaby zrobić tak, żeby nie powtarzać tego samego kodu. Wyciągnij to, czym te funkcje się różnią do argumentu, a z reszty zrób powtarzalną funkcję, która obsłuży wszystkie trzy zapytania.